content/docs/command-reference/plots/index.md

```diff
@@ -20,62 +20,45 @@ positional arguments:
 ## Description
 
 DVC provides a set of commands to visualize _continuous metrics_ of machine
-learning experiments. Usual examples of plots are AUC curves, loss functions,
-and confusion matrices.
+learning experiments (see []()). Usual examples of plots are AUC curves, loss
+functions, and confusion matrices.
 
 These metrics represent plots, and should be stored as data series in one of the
-supported [file formats](#file-formats). Metric files are created by users, or
-generated by their modeling or data processing code.
+[supported file formats](#supported-file-formats). metric files are created by
+users, or generated by their modeling or data processing code.
 
-The plot commands can work with metric files that are committed to a repository
-history, data files controlled by DVC, or any other file in the
-<abbr>workspace</abbr>. For example, the command `dvc plots diff` generates a
-plot with two versions of a metric:
-
-```dvc
-$ dvc plots diff -d logs.csv
-file:///Users/dmitry/src/plot/logs.html
-```
-
-![](/img/plots_auc.svg)
+`dvc plots` subcommands can work with metric files committed to a Git repo
+history, data files controlled by DVC, or any other file in file system.
 
 DVC generates plots as HTML files that can be open with a web browser. These
 HTML files use [Vega-Lite](https://vega.github.io/vega-lite/). Vega is a
 declarative grammar for defining plots using JSON. The plots can also be saved
 as SVG or PNG image filed from the browser.
 
-### Difference between continuous and scalar metrics
+### Types of metrics
```

Comment:
same here - should go before description

Reply:
Will reply in https://github.com/iterative/dvc.org/pull/1360#pullrequestreview-421446357.